spec/requests/api/v1/users/create_request_spec.rb

```diff
@@ -53,10 +53,11 @@
          "Zip code can't be blank",
          'Zip code is not a number',
          "Summary can't be blank",
+         "Goal can't be blank",
          "Full name can't be blank"]
       end
 
-      before { post '/api/v1/users', params: { goal: 'value1' } }
```

Comment:
We changed this edge case... params can only come into this route from the form select values from FE, so the logic was that if no param provided (blank form field submission), we get back the model validations error messages..

Reply:
I like this update a lot!  Makes sense to me!